plugins/modules/aci_interface_policy_pim.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_policy_pim
+short_description: Manage Protocol-Independent Multicast (PIM) interface policies (pim:IfPol)
+description:
+- Manage Protocol Independent Multicast interface policies for Tenants on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of an existing Tenant.
+    type: str
+    aliases: [ tenant_name ]
+  pim:
+    description:
+    - The PIM interface policy name.
+    - The name cannot be changed after the object has been created.
+    type: str
+    aliases: [ pim_interface_policy, name ]
+  authentication_key:
+    description:
+    - The authentication key.
+    type: str
+    aliases: [ auth_key ]
+  authentication_type:
+    description:
+    - The authentication type.
+    type: str
+    choices: [ none, md5_hmac ]
+    aliases: [ auth_type ]
+  control_state:
+    description:
+    - The PIM interface policy control state.
+    - 'This is a list of one or more of the following controls:'
+    - C(multicast_domain_boundary) -- Boundary of Multicast domain.
+    - C(strict_rfc_compliant) -- Only listen to PIM protocol packets.
+    - C(passive) -- Do not send/receive PIM protocol packets.
+    type: list
+    elements: str
+    choices: [ multicast_domain_boundary, strict_rfc_compliant, passive ]
+  designed_router_delay:
```

Comment:
```suggestion
  designated_router_delay:
```
I think it is designated not designed. Please change all occurrences.